internal/pkg/rotator/aws.go

```diff
@@ -11,8 +11,7 @@ import (
 )
 
 type Group struct {
-	launchConfigurationName string
```

Comment:
We did not use this, so no need to even store it and handle whether we're working with launch configs or templates